src/Components/Card.js

```diff
@@ -23,7 +23,8 @@ const Card = ({
   origin,
   location,
   mode,
```

Comment:
The mode prop does not describe what is is being used for. You should rename it to be more descriptive.